Sources/BraveWallet/Extensions/BraveWalletExtensions.swift

```diff
@@ -342,6 +348,23 @@ extension BraveWallet.CoinMarket {
   }
 }
 
+extension BraveWallet.KeyringId {
+  static func keyringId(for coin: BraveWallet.CoinType, _ chainId: String) -> BraveWallet.KeyringId {
```

Comment:
nit (keep 2nd param name, `on` might work too just so it's a bit more descriptive): 
```suggestion
  static func keyringId(for coin: BraveWallet.CoinType, chainId: String) -> BraveWallet.KeyringId {
```